### PR TITLE
add arch specific machine types to csv-merger

### PR DESCRIPTION
**Pull request by tiraboschi**
**What this PR does / why we need it**:
add optional parameters to be able to specify
arch specific machine types to the csv-merger tool.

Follow-up on #2863

**Reviewer Checklist**

> Reviewers are supposed to review the PR for every aspect below one by one. To check an item means the PR is either "OK" or "Not Applicable" in terms of that item. All items are supposed to be checked before merging a PR. 

- [ ] PR Message
- [ ] Commit Messages
- [ ] How to test
- [ ] Unit Tests
- [ ] Functional Tests
- [ ] User Documentation
- [ ] Developer Documentation
- [ ] Upgrade Scenario
- [ ] Uninstallation Scenario
- [ ] Backward Compatibility
- [ ] Troubleshooting Friendly

**Jira Ticket**:

```jira-ticket
https://issues.redhat.com/browse/CNV-37722
```

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tools/csv-merger/csv-merger.go [hunk 1]
```diff
@@ -534,6 +536,8 @@ func getDeploymentParams() *components.DeploymentOperatorParams {
 		VirtIOWinContainer: *kvVirtIOWinImage,
 		Smbios:             *smbios,
 		Machinetype:        *machinetype,
+		Amd64MachineType:   *amd64MachineType,
+		Arm64MachineType:   *arm64MachineType,
```
Comment (machadovilaca) [inline tools/csv-merger/csv-merger.go:540]: You are adding the `Amd64MachineType` and `Arm64MachineType`, but I don't see them defined in https://github.com/tiraboschi/hyperconverged-cluster-operator/blob/3585d39f79f59024d7b1b3fef11055b5ef99ab3f/pkg/components/components.go#L65

Comment (machadovilaca) [reply tools/csv-merger/csv-merger.go:540]: I don't know to which version GitHub took me, but I do see them on `main`

Comment (tiraboschi) [reply tools/csv-merger/csv-merger.go:540]: They come with: https://github.com/tiraboschi/hyperconverged-cluster-operator/commit/35a84c89be94acfa43e8a867a9f0a169d0a8d3d0#diff-f1531df30500812ec42a33a52023c9ceca8a93d4972132fd241ad143b107c063R79-R80

**Pull request merged**